src/app/help/faq/faq.component.ts

```diff
@@ -1,9 +1,8 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { HelpService } from '../../shared/services/help.service';
 import { BaseComponent } from '../../base.component';
 import { Title } from '@angular/platform-browser';
-import * as $ from 'jquery';
```

Comment:
Looks like you removed jquery, however the search functional (line 90) is still expecting jquery,